src/custom.tex

```diff
@@ -1347,274 +1358,1121 @@ \subsection{\LaTeX{} Units}\index{units (\TeX)}\index{dimensions}%
+using ths \csi{indent} command everywhere, simply use the \pai*{indentfirst}
+package in your preamble.
+\begin{example}[standalone, paperheight=2.5cm, paperwidth=0.5\linewidth]
+\usepackage{indentfirst}
+% ...
 
-Whenever possible, I avoid using absolute lengths in
-\LaTeX{} documents. I rather try to base things on the width or height
-of other page elements. For the width of a figure this could
-be \verb|\textwidth| in order to make it fill the page.
+\begin{document} %!hide
+\section{Title}
 
-The following 3 commands allow you to determine the width, height, and
-depth of a text string.
+The first paragraph is now
+indented.
+\end{document} %!hide
+\end{example}
 
+To create a non-indented paragraph, use \csi{noindent} as the first command of
+the paragraph. This might come in handy when you start a document with body
+text and not with a sectioning command.
+
+\subsection{Page Layout}\label{sec:page_layout}
+\begingroup
+\LshortExampleSetup{
+  standalone,
+  template=empty,
+  paperwidth=6cm,
+}
+As you have seen, \LaTeX{} allows you to specify the \wi{paper size} via
+options in the \cs{documentclass} command. It then automatically picks the
+right text \wi{margins}, but sometimes you may not be happy with the predefined
+values. Naturally, you can customize them to your liking. But before you start
+making the margins as narrow as possible to cram the text in, take a few
+seconds to think. As with most things in \LaTeX, there are good reasons for the
+page layout to be as it is.
+
+Sure, compared to your off-the-shelf page of your favourite WYSIWYG editor, the
+margins look awfully wide. But take a look at your favourite, professionally
+printed book and count the number of characters on a standard text line. You
+will find that most lines contain between 45 and 80 characters. Now do the same
+on your \LaTeX{} page. You will find that the same relationship holds.
+Empirical studies suggest that averaging around 66 characters per line creates
+the optimal reading experience for readers. If you want to save space when
+printing your document consider using \cargv{twocolumn} option or smaller page
+sizes.
+
+With that warning in place let us proceed to the proper introduction of
+\pai*{geometry} package that allows you to easily customize the page dimensions
+of your document. It is worth noting that simply including the package in your
+preamble will result in considerably narrower margins, (the very thing we
+warned you to avoid) so only use the package if you intend to set them
+manually or use the \cargv{pass} option that disables most of the package
+functions but retains the paper size adjustments.
+
+As we have mentioned in \autoref{options}, simply setting \cargv{a5paper} (or
+similar) option will only adjust margins of the document without changing the
+paper dimensions themselves. The simplest way to fix that is to include the
+\pai{geometry} package with your preamble. It will read the page size option
```

Comment:
```suggestion
paper dimensions themselves. The simplest way to fix that is to add the
\pai{geometry} package to your preamble. It will read the page size option
```